transport/src/test/java/io/netty/channel/pool/FixedChannelPoolTest.java

```diff
@@ -297,6 +305,12 @@ public void run() {
         pool.release(channel).syncUninterruptibly();
         sc.close().syncUninterruptibly();
         channel.close().syncUninterruptibly();
+
+        ChannelPool.Stats stats = pool.stats();
+        assertEquals(0, stats.leasedChannelsCount());
+        assertEquals(0, stats.pendingChannelsCount());
+        // TODO: (need to wait the close operation is completed. how?
+//        assertEquals(0, stats.availableChannelsCount());
```

Comment:
I think for 4.1 we could add a new method to the interface like `asyncClose()` and make the "normal" `close()` blocking. WDYT ?